R/gen_requirements_helpers.R

```diff
@@ -151,5 +155,7 @@ write_requirements_file = function(package_requirements, file_path="requirements
   #'
   #' @param package_requirements Character vector of requirements to write to file.
   #' @param file_path Path to write requirements to.
-  writeLines(package_requirements, file_path)
+  requirements_file_contents = c(AUTO_GEN_COMMENTS, package_requirements)
```

Comment:
I think if `package_requirements` is empty there should be a note displayed letting the user know it was empty. It might be a lazy way of initializing the text file in a new project for someone (or part of a standard script that's run to ensure the file exists at all)

Reply:
Opinion on whether the note should be a `warning` or a `message`?